vendor/github.com/golang/glog/glog.go

```diff
@@ -676,10 +676,7 @@ func (l *loggingT) output(s severity, buf *buffer, file string, line int, alsoTo
 		}
 	}
 	data := buf.Bytes()
-	if !flag.Parsed() {
```

Comment:
I don't like it either. But to do a fork of glog, I don't know ...I'd rather use another libraries